app/src/main/java/yuricfurusho/traderev/photos/Photo.kt

```diff
@@ -0,0 +1,5 @@
+package yuricfurusho.traderev.photos
+
+data class Photo(
+    var url: String = ""
+)
```

Comment:
this is just for simplicity now, it will change to the actual fields we want to use from Unsplash library